portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * chose a random element from an array
+ */
+function randomFromArray(arr){
+  randomChoice = Math.floor(Math.random() * arr.length)
+  console.log(randomChoice + "/" + arr.length)//checks that the choice is indeed with in the constraints
```

Comment:
If you no longer need it, remove the logging.

Reply:
If you want to leave it there for now, that's ok. Please move comment above the line instead of after it.

Reply:
Sounds good. I've removed it since I won't be needing it.